### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by rajnavadiya972**
feat-User register and User login

### CODE REVIEW

**Review by th-jay — CHANGES_REQUESTED** [review]
- remove unnecessary code and consoles.
- Queries should be in the proper folder, not in a model folder.
- for error throws and error with proper message not just with only consoles.

File: index.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
-const express = require("express");
-require("dotenv").config();
-const { client } = require("./src/config/database.js");
+import express from "express";
+import { } from "dotenv/config";
```
Comment (th-jay) [inline index.js]: justify this

File: src/config/database.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
-const { Client } = require("pg");
+import pg from "pg";
+
+const { Client } = pg;
```
Comment (th-jay) [inline src/config/database.js]: simplify this

**Review by th-jay — CHANGES_REQUESTED** [review]
Previous comments are still not resolved.

File: index.js (hunk 1 above)
Comment (rajnavadiya972) [reply index.js]: ok i understand {} from not require.
i made changes its only  import "dotenv/config"

File: src/config/database.js (hunk 2 above)
Comment (rajnavadiya972) [reply src/config/database.js]: when i simplify this it thow an error
import { Client } from 'pg'
         ^^^^^^
SyntaxError: Named export 'Client' not found. The requested module 'pg' is a CommonJS module, which may not support all module.exports as named exports. 
CommonJS modules can always be imported via the default export, for example using:
import pkg from 'pg';
const { Client } = pkg;

**Pull request closed without merging**